shared/player/ytPlayer.js

```diff
@@ -131,6 +131,9 @@ window.onYouTubePlayerAPIReady = () => {
       onStateChange(e) {
         yt.setState(e.data);
       },
+      onError() {
```

Comment:
perhaps checking the type of the error can solve the false-positive problem.

Reply:
`2` – The request contains an invalid parameter value. For example, this error occurs if you specify a video ID that does not have 11 characters, or if the video ID contains invalid characters, such as exclamation points or asterisks.
`5` – The requested content cannot be played in an HTML5 player or another error related to the HTML5 player has occurred.
`100` – The video requested was not found. This error occurs when a video has been removed (for any reason) or has been marked as private.
`101` – The owner of the requested video does not allow it to be played in embedded players.
`150` – This error is the same as 101. It's just a 101 error in disguise!

so will apply only if `>= 100`

Reply:
The false positive problem has something to do with the `Dev` configuration in the server. This is a behavior that I encounter before, I never paid attention to it because it didn't get in the way of testing other stuff, but now it does.